templates/data/postgresql/support.html

```diff
@@ -0,0 +1,731 @@
+{% extends 'base_index.html' %}
+
+{% block meta_copydoc %}
+https://docs.google.com/document/d/1scPktUiI2XRVoYcR4L6qUZ91iZ0upZA-FaqyJN7uMJg/edit
+{% endblock meta_copydoc %}
+
+{% block title %}Get comprehensive support for PostgreSQL, the world's most acclaimed open source database management system.{% endblock %}
+
+{% block meta_description %}
+Get support for PostgreSQL from Canonical, the experts in open-source. We provide ticket, phone or screen sharing support 24/7 for up to ten years per release track.
+{% endblock %}
+
+{% block body_class %}
+   is-paper
+{% endblock body_class %}
+
+{% block content %}
+<section class="p-section--hero">
+  <div class="row--25-75-on-large">
+    <div class="col">
+      <div class="p-section--shallow">
+        {{ image (
```

Comment:
please remove the space between "image" and the bracket and format the file with djLint

Reply:
Fixed